### DISCUSSION TITLE
How many repositories should my language be in to be detected by Linguist?

### DISCUSSION THREAD

**Discussion by sparkles-laurel** [github-linguist/linguist#5322] [Q&A] [answered]
The contribution guidelines say:
> We try only to add languages once they have some usage on GitHub. In most cases we prefer that each new file extension be in use in at least 200 unique :user/:repo repositories before supporting them in Linguist.

Does that mean,

a) my language should be in 200 unique repositories,
b) my language should be in one repository of 200 unique users
c) something another?

Please forgive me if I asked this question rudely.

**Comment by lildude** [comment, marked as answer]
> a) my language should be in 200 unique repositories,
> b) my language should be in one repository of 200 unique users

A combination of these... with some validation to ensure peeps aren't gaming the system by creating 200 repos under their own username 😁 and to ensure all the usage isn't simply attributed to a fork of a single repo.

**Reply by sparkles-laurel** [reply]
OK, thanks.
